en/guide/overriding-express-api.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: Overriding the Express API
+menu: guide
+lang: en
+---
+<div id="page-doc" markdown="1">
+
+# Overriding the Express API
+
+Since the Express API are extensions to Node's HTTP request and response objects, if required, you can modify the behavior of some of the existing methods and properties.
+
+## Methods
+
+You can override the signature and behavior of existing methods with your own, by assigning a custom function.
+
+Following is an example of overriding the behavior of [res.sendStatus](/{{ page.lang }}/4x/api.html#res.sendStatus).
+
+```js
+app.response.sendStatus = function (statusCode, type, message) {
+  // code is intentionally kept simple for demonstration purpose
+  return this.contentType(type)
+  .status(statusCode)
+  .send(message);
+}
+```
+
+The above implementation completely changes the original signature of [res.sendStatus](/{{ page.lang }}/4x/api.html#res.sendStatus). It now accepts a status code, encoding type, and the message to be sent to the client.
+
+The overridden method may now be used this way:
+
+```js
+res.sendStatus(404, 'application/json', '{"error":"resource not found"}')
+```
+
+## Properties
+
+Properties in the Express API are either:
+
+1. **Inherited from the Node core** - `res.headersSent` etc.
+2. **Assigned properties** - `req.app`, `req.baseUrl`, `req.originalUrl`, `res.app`, etc.
```

Comment:
Yea, sorry. I was saying that they are properties that are on the prototype, not assigned properties as written here. That means that someone actually _can_ override them, and the "buy why??" part was just wondering why someone would want to--which was unnecessary commentary :)